meta-directive.js

```diff
@@ -0,0 +1,35 @@
+/*!
+ * Copyright (c) 2016 Digital Bazaar, Inc. All rights reserved.
+ */
+define([], function() {
+
+'use strict';
+
+function register(module) {
+  module.directive('brMeta', factory);
+}
+
+/* @ngInject */
+function factory() {
+  return {
+    replace: true,
+    restrict: 'E',
+    templateUrl: requirejs.toUrl('bedrock-angular-meta/meta-directive.html'),
+    controller: Ctrl,
+    controllerAs: '$ctrl'
+  };
+
+  /* @ngInject */
+  function Ctrl($rootScope, brMetaService, $location) {
+    var self = this;
+    self.meta = brMetaService.meta[$location.url()];
```

Comment:
While it's true that the event method is not a complete solution, here's what I have found.

I have not been able to find anything like $rootScope.route that contains the route path.  If you know details, please let me know.  I believe that 
```
$rootScope.$on('$routeChangeSuccess', function() { path = $location.url()})
```
 is the generally accepted solution.  And there was an iteration that involved that.  The problem with that solution (and others) is that brMetaService.meta[foo] is only initialized after the controller for that route has initialized and run brMetaService.set().  My understanding is that the controllers are initialized after the `$routeOnChangeSuccess` event, which also matches my observations.

So, we need a way for the `<br-meta>` directive to act on a route change only after the controller for the route has initialized.

It may be that a route resolver will be required to collect the information for the meta tags and then utilize an $routeChangeSuccess handler in `<br-meta>`

Also, the existing implementation works well as long as ALL routes call `brMetaService.set()` which of course is not what we want.  A route/controller combo that does not call `brMetaService.set()` will not trigger meta tag changes.